light/client.go

```diff
@@ -64,11 +64,10 @@ func SkippingVerification(trustLevel tmmath.Fraction) Option {
 	}
 }
 
-// PruningSize option sets the maximum amount of headers & validator set pairs
-// that the light client stores. When Prune() is run, all headers (along with
-// the associated validator sets) that are earlier than the h amount of headers
-// will be removed from the store. Default: 1000. A pruning size of 0 will not
-// prune the light client at all.
+// PruningSize option sets the maximum amount of light blocks that the light
```

Comment:
should this be configurable through the cmd line?

Reply:
yeah I think that would make sense